yellowbrick/features/histogram.py

```diff
@@ -0,0 +1,202 @@
+        Parameters
+        ----------
+        X : ndarray or DataFrame of shape n x 1
+            A matrix of n instances with 1 feature
+        kwargs : dict
+            keyword arguments passed to Scikit-Learn API.
+        """
+
+        #throw an error if X has more than 1 column
+        if is_dataframe(X):
+            nrows, ncols = X.shape
+
+            if ncols > 1:
+                raise YellowbrickValueError((
+                    "X needs to be an ndarray or DataFrame with one feature, "
+                    "please select one feature from the DataFrame"
+                ))
+
+        # Handle the feature name if it is None.
+        if self.feature is None:
+
+            # If X is a data frame, get the columns off it.
+            if is_dataframe(X):
+                self.feature = X.columns
+
+            else:
+                self.feature = ['X']
+
+        self.draw(X)
+        return self
+
+
+    def poof(self, **kwargs):
+        """
+        Creates the labels for the feature and target variables
+        """
+
+        self.ax.set_xlabel(self.feature)
+        self.ax.set_ylabel(self.target)
+        self.finalize(**kwargs)
+
+    def finalize(self, **kwargs):
+        """
+        Finalize executes any subclass-specific axes finalization steps.
+        The user calls poof and poof calls finalize.
+        Parameters
+        ----------
+        kwargs: generic keyword arguments.
+        """
+
+        for tk in self.ax.get_xticklabels():
+            tk.set_visible(True)
+            
+        for tk in self.ax.get_yticklabels():
+            tk.set_visible(True)
+        
+        
+##########################################################################
+## Quick Method
+##########################################################################
+        
+def balanced_binning_reference(ax=None, feature=None, target='Counts', bins=4, **kwargs):
+    
+    """
+    BalancedBinningReference allows to generate a histogram with vertical lines
+    showing the recommended value point to bin your data so they can be evenly
+    distributed in each bin.
+
+    Parameters
+    ----------
+    ax : matplotlib Axes, default: None
+        This is inherited from FeatureVisualizer and is defined within
+        BalancedBinningReference.
+    feature : string, default: None
+        The name of the X variable
+        If a DataFrame is passed to fit and feature is None, feature
+        is selected as the column of the DataFrame.  There must be only
+        one column in the DataFrame.
+    target : string, default: None
+        The name of the Y variable
+        Default: Counts
+    bins : number of bins to generate the histogram
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    """
+    
+    # Initialize the visualizer
+    visualizer = balanced_binning_reference(ax=ax, X, y, bins=bins)
```

Comment:
The tests are currently failing on this line of code, positional arguments have to come before keyword arguments. I would recommend just making them all keyword arguments:

```python
visualizer = balanced_binning_reference(ax=ax, X=X, y=y, bins=bins)
```

That way changing the order of the arguments won't break the quick function.